docs/content/theme/theming-for-older-browsers.md

```diff
@@ -9,40 +9,33 @@ menu = "theme"
 tags = [ "theme" ]
 +++
 
-# Non-Supported Browsers
 
-
-
-## Microsoft Edge
-
-Though modern, Edge still does not support Shadow DOM... @TODO
-
-
-## Microsoft Internet Explorer 11 
+## Microsoft Internet Explorer 11
 
 This is an attempt to document the things that usually go sideways with IE11.
 
-	<!-- TODO inline image-->
-
-* Using the CSS property `:focus-within` breaks the styles polyfill  
-
+* Using the CSS property `:focus-within` breaks the styles polyfill (ShadyCSS) but you won't see any helpful messages in the rendered page. It will just fail to load entirely.
 
-* The IE11 polyfill can interpret nested vars, but only on the host: 
+* The IE11 polyfill can interpret nested variables, but only on the host:
 
-    ```
+    ```css
     :host {
        padding: var(--ie-fake-var, var(--another, 1em));
     }
     ```
 
-   will result in  
- 
-	```
-	.pfe-cta-1 {
-	    padding: 1em;
-	}
-	```
+   will result in:
+
+    ```css
+    .pfe-cta-1 {
+        padding: 1em;
+    }
+    ```
+
+    You can override either of the two provided variables, `--ie-fake-var` or `--another`.
+
+* IE11 will ignore slotted styles, it doesnt matter if you style it with `::slotted()`, `slot::slotted()` or even if you add a class to the children with JavaScript & try to style that. IE11 ignores all of it because of how it is restructuring the DOM using ShadyDOM.
 
-    and you can override either of those variables!
+## Safari
 
-2. IE11 will ignore slotted styles, it doesnt matter if you style it with `::slotted()` `slot::slotted()` or even if you add a class to the children with JavaScript & try to style that. IE11 ignores all of it.
+@TODO: Add some notes here on nuances with Safari
```

Comment:
flagging this in case you wanted to complete it before merging

Reply:
Thanks, that's why I added the TODO, so it could be returned to later but not now because it's not urgent.